cudaextender/data/sample_scored_segment_pairs.csv

```diff
@@ -1375,3 +1033,305 @@
 249548,249548,71,6601
 249737,249737,33,3187
 249960,249960,39,3655
+207793,207753,119,3189
```

Comment:
Sorry I missed the test data change portion of the discussion. Was just the order changed? In that case, can you sort one of these (read scored_segment_pairs or output scored_segment_pairs) before equality checking?

Reply:
There have been changes to the data too. During the development after generating this first sample, I resolved the case where there are multiple segment pairs that differ by 1 in the positions but should they should map to the same final segment pair. For example,

```
24377,24377,202,18237
24377,24377,201,18146
24377,24377,200,18046
24377,24377,199,17955
24377,24377,198,17864
```